question11/main.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// loading img 
+const wrapper = document.getElementById('wrapper');
+const loadImg = document.createElement('img');
+const showLoadImg = () => {
+    loadImg.src = "loading-circle.gif";
+    wrapper.appendChild(loadImg);
+} 
+
+async function callApi() {
+    showLoadImg();
+    try {
+        const res = await fetch('https://jsondata.okiba.me/v1/json/dFQLo210903231659');
+        const json = await res.json();
+        return json.data;
+    }
+    catch(error) {
+        wrapper.textContent = "データが取得できませんでした";
+        console.error(error);
+        return null;
+    }
+    finally{
+        loadImg.remove()
+    }}
+
+    async function init(){
+        const result = await callApi();
+        if (result !== null){
+            createList(result);
+        }
```

Comment:
nits
Empty characters, "0", null and undefined will be evaluated as false, so you can also do this!
（I learned it from Mr. Morita.）

```suggestion
        if (result){
            createList(result);
        }
```

Reply:
Thank you for the review! This is really important thing! I fixed it that way. Please confirm.

Reply:
Q
Why did you remove the return null?💦

Reply:
Sorry! I thought 『empty characters, "0", null, and undefined are evaluated as false』 meant that they don't have to be "null". Now, "undefined" is passed in here, and is considered false. Should I specify “null” here??

Reply:
Thank you for your reply! I understand now.